docs/enterprise-guide/data-sandboxes.md

```diff
@@ -136,15 +136,15 @@ I.e., if a user belongs to two user groups, both of which have been given sandbo
 
 You can't currently create data sandboxes with non-SQL databases such as Google Analytics, Apache Druid, or MongoDB .
 
-#### Data sandboxes do not work on SQL/native queries
+#### Data sandboxes do not work if users can execute SQL/native queries
```

Comment:
```suggestion
#### Data sandboxes don't work for groups that can execute SQL/native queries
```
Good distinction, I just substituted groups so we avoid the term "users" and emphasize that permissions are granted to groups, not people.